src/app/redux/reducers/identity.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+export interface IdentityState {
+  address: string | null
+  chainId: number | null
+}
+
+interface ReceiveIdentityPayload {
+  address: string
+}
+
+interface ReceiveChainIdPayload {
+  chainId: number
+}
+
+export const initialState: IdentityState = {
+  address: null,
+  chainId: null
+}
+
+const identitySlice = createSlice({
+  name: 'identity',
+  initialState,
+  reducers: {
+    receiveIdentity (state: IdentityState, { payload: { address } }: PayloadAction<ReceiveIdentityPayload>) {
```

Comment:
I would say 
```suggestion
    changeIdentity (state: IdentityState, { payload: { address } }: PayloadAction<ReceiveIdentityPayload>) {
```

Reply:
Are we planning to extend this method? If not I would call it change account, not identity